blueprint/index.md

```diff
@@ -0,0 +1,164 @@
+* [Configure your EventBridge software as a service (SaaS) integration](#configure-your-eventbridge-software-as-a-service-saas-integration "Goes to the Configure your EventBridge software as a service (SaaS) integration section")
+* [Edit the configuration files](#edit-the-configuration-files  "Goes to the Edit the configuration files section")
+* [Deploy the application](#deploy-the-application "Goes to the Deploy the application section")
+* [Delete an OAuth Client](#delete-an-oauth-client "Goes to the Delete an OAuth Client section")
+* [Running locally](#running-locally "Running locally")
+
+### Clone the repository containing the project files
+
+Clone the [aws-eventbridge-oauth-client-delete-blueprint](https://github.com/GenesysCloudBlueprints/aws-eventbridge-oauth-client-delete-blueprint "Opens the aws-eventbridge-oauth-client-delete-blueprint repository in GitHub") repository from GitHub.
+
+### Enable the Amazon EventBridge integration in your Genesys Cloud account
+
+This step is only necessary if the Amazon EventBridge integration is not yet enabled in your Genesys Cloud account.  
+
+Follow the steps in [About the Amazon EventBridge integration](https://help.mypurecloud.com/articles/about-the-amazon-eventbridge-integration/ "Opens the About the Amazon EventBridge integration on the Genesys Cloud Resource Centre").
+
+### Configure your EventBridge software as a service (SaaS) integration
+
+Configure your [EventBridge software as a service (SaaS)](https://console.aws.amazon.com/events/home?region=us-east-1#/partners) integration, and note the event source name (e.g., aws.partner/example.com/1234567890/test-event-source). Before proceeding, ensure that your event source is listed as Pending.
+
+### Edit the configuration files
+
+Edit `src/python/config.py` and `src/typescript/src/config.ts` and add a PagerDuty API token.  
+
+Feel free to remove the source code of either Lambda function and remove the references to it from `template.yml` if you don't wish to use it.
+
+### Deploy the application
+
+From the repo root, run the following command:
+
+```
+sam build
+```
+
+Followed by:
+
+```
+sam deploy --guided
+```
+
+:::primary
+You must be authenticated to use the CLI before running the above command
+:::
+
+Choose an appropriate Stack Named when prompted.  
+The parameter `EventSourceName` must be the event source name noted from the [Configure your EventBridge software as a service (SaaS) integration](#configure-your-eventbridge-software-as-a-service-(saas)-integration "Goes to the Configure your EventBridge software as a service (SaaS) integration section") step.
+
+### Delete an OAuth Client
+
+If you have the permissions to create and delete OAuth clients in Genesys Cloud, use the following guide to [Create an OAuth client](https://help.mypurecloud.com/articles/create-an-oauth-client/). Then, delete the OAuth Client in order to trigger the Lambda functions.  
```

Comment:
Add a warning here begging people not to shoot themselves in the foot by deleting a real oauth client they're using in production. That should be obvious, but good to call it out explicitly. Good call on specifically saying to create one to delete!

One time I saw a customer's call center go down because someone borrowed the ethernet cable to their SQL server to hook up their desktop. Mistakes can, and will, happen. :)